src/utils/app_utils.py

```
@@ -51,247 +59,180 @@ def get_ip_address():
-        # Enforce maximum upload size (bytes). Default 10 MB; override with env MAX_UPLOAD_BYTES
-        try:
-            max_upload_bytes_env = os.getenv("MAX_UPLOAD_BYTES")
-            max_upload_bytes = (
-                int(max_upload_bytes_env) if max_upload_bytes_env else 10 * 1024 * 1024
-            )
-        except Exception:
-            max_upload_bytes = 10 * 1024 * 1024
+        # Read raw bytes once so this works with both FileStorage and test fakes.
+        content = file.read()
+        if content is None:
+            raise RuntimeError("Empty upload content")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)